src/TreeView/TreeView.tsx

```diff
@@ -405,6 +408,40 @@ const Item: React.FC<TreeViewItemProps> = ({
   )
 }
 
+/** Lines to indicate the depth of an item in a TreeView */
+const LevelIndicatorLines: React.FC<{level: number}> = ({level}) => {
+  return (
+    <Box sx={{width: '100%', display: 'flex'}}>
+      {Array.from({length: level - 1}).map((_, index) => (
+        <Box
+          key={index}
+          sx={{
+            width: '100%',
+            height: '100%',
+            transition: 'border-color 33.333ms linear',
```

Comment:
A `33.333ms` transition appears imperceptible to me. I'd recommend starting without a transition, and coming back to refining UI animations later.

Reply:
I copied this from the ActionList styles but I'm happy to remove transitions for now 👍